packages/core/src/components/Shadows/EditShadowModal.tsx

```diff
@@ -357,19 +357,26 @@ export function EditShadowModal({
                 }}
               >
                 {newInitialValues?.map((_arg: ShadowValue, index: number) => {
+                  const opacity = index === shadowIndex ? '1' : '0.4';
```

Comment:
Using an opacity based approach because it's the same one used by ChakraUI - It adds an `opacity: 0.4` for disabled inputs.